src/_data/blog-dates.json

```diff
@@ -0,0 +1,432 @@
+{
+    "2014-01-19-breaking-change-formatter.md": "2022-02-05T06:13:45.000Z",
+    "2014-01-20-breaking-change-config-file.md": "2022-02-05T06:13:45.000Z",
+    "2014-01-20-eslint-0.3.0-released.md": "2022-02-05T06:13:45.000Z",
+    "2014-02-12-eslint-0.4.0-released.md": "2022-02-05T06:13:45.000Z",
+    "2014-03-03-eslint-0.4.2-released.md": "2022-02-05T06:13:45.000Z",
+    "2014-03-18-eslint-0.4.3-released.md": "2022-02-05T06:13:45.000Z",
+    "2014-03-25-eslint-0.4.4-released.md": "2022-02-05T06:13:45.000Z",
+    "2014-03-29-eslint-0.4.5-released.md": "2022-02-05T06:13:45.000Z",
+    "2014-04-10-eslint-0.5.0-released.md": "2022-02-05T06:13:45.000Z",
+    "2014-04-17-eslint-0.5.1-released.md": "2022-02-05T06:13:45.000Z",
```

Comment:
Looks like blog posts copied over from the old website repo (https://github.com/eslint/archive-website) will have the update date `2022-02-05`, which I think would be misleading. Perhaps we could get the last updated date from the old website repo, or maybe just use the publish date as the last updated date for these posts?

Reply:
Good catch. :+1: to using the publish date as the last updated date.

Reply:
I see, there was a commit https://github.com/eslint/archive-website/commit/5aa1575a37e3b4a381b7f5f57acf31f59ee45947 that fixed the authors on that date.

Reply:
Now, I think this raises a question: Does any change to the blog post's source file constitute an update worth showing in the blog post? I think readers would be interested in factual changes to the content, not e.g.,  authors, labels, etc.., maybe not even fixing small typos in the text. Since we rarely update the content of blog posts, I'd be in favor of manually updating this date rather than automatically in pre-commit.

Reply:
The problem with manual updates is that they tend not to happen. To me, the fact that we rarely update blog posts is a good argument for automatically updating the "last updated date" because we're more likely to forget.

Reply:
@mdjermanovic what do you think?

Reply:
Okay, we can always undo the date change directly in the PR if we don't want to modify the date based on the kind of change in the blog post file.